sos/plugins/nfsganesha.py

```diff
@@ -0,0 +1,50 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Please fix the subject format ([Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)).